src/Controls/src/Xaml/MarkupExpressionParser.cs

```diff
@@ -122,12 +123,13 @@ internal static bool MatchMarkup(out string match, string expression, out int en
 			return true;
 		}
 
+#if !__SOURCEGEN__
 		[RequiresUnreferencedCode(TrimmerConstants.XamlRuntimeParsingNotSupportedWarning)]
+#endif
 		protected Property ParseProperty(IServiceProvider serviceProvider, ref string remaining)
```

Comment:
Confused by use of `__SOURCEGEN__` here.

If we only ship a single `Microsoft.Maui.Xaml.dll`, are we going to ship 2 copies now? One with `__SOURCEGEN__` defined and one not defined?